nestjs/src/courses/course-tasks/course-tasks.service.ts

```diff
@@ -99,6 +100,7 @@ export class CourseTasksService {
   }
 
   public updateCourseTask(id: number, courseEvent: Partial<CourseTask>) {
+    console.log(courseEvent);
```

Comment:
Are you sure we need this `console.log`?

Reply:
thanks